website/docs/r/api_management_api.html.markdown

```diff
@@ -0,0 +1,124 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_api_management_api"
+sidebar_current: "docs-azurerm-resource-api-management-api-x"
+description: |-
+  Create a API Management API.
+---
+
+# azurerm_api_management_api
+
+Create a API Management API component.
+
+## Example Usage (import from Open API spec)
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "api-rg-dev"
+  location = "West Europe"
+}
+
+resource "azurerm_api_management" "test" {
+  name                = "api-mngmnt-dev"
+  publisher_name      = "My Company"
+  publisher_email     = "company@terraform.io"
+  sku {
+    name     = "Developer"
+    capacity = 1
+  }
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+
+resource "azurerm_api_management_api" "test" {
+  name                = "conferenceapi"
+  service_name        = "${azurerm_api_management.test.name}"
+  path                = "conference"
+  import {
+    content_format = "swagger-link-json"
+    content_value  = "http://conferenceapi.azurewebsites.net/?format=json"
+  }
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the API Management API.
+
+* `resource_group_name` - (Required) The Name of the Resource Group where the API Management API exists.
+
+* `location` - (Required) The Azure location where the API Management API exists.
+
+* `service_name` - (Required) The Name of the API Management Service where the API Management API exists
+
+* `path` - (Required) Relative URL uniquely identifying this API and all of its resource paths within the API Management service instance. It is appended to the API endpoint base URL specified during the service instance creation to form a public URL for this API.
+
+---
+
+* `service_url` - (Optional) Absolute URL of the backend service implementing this API.
+
+* `description` - (Optional) Description of the API. May include HTML formatting tags.
+
+* `protocols` - (Optional) A list of protocols the operations in this API can be invoked. Supported values are `http` and `https`. Default is `https`.
+
+* `subscription_key_parameter_names` - (Optional) A `subscription_key` block as documented below.
+
+* `import` - (Optional) A `import` block as documented below.
+
+* `soap_pass_through` - Make API Management expose a SOAP front end, instead of a HTTP front end.
+
+---
+
+A `subscription_key_parameter_names` block exports the following:
```

Comment:
whilst this is Computed (and thus technically exported) - it's optional/computed, as such can we update this to match the other blocks:

```suggestion
A `subscription_key_parameter_names` block supports the following:
```